projects/storefrontlib/src/cms-components/configurator/generic/configure-cart-entry/configure-cart-entry.component.ts

```diff
@@ -40,5 +43,19 @@ export class ConfigureCartEntryComponent {
     return this.readOnly;
   }
 
-  constructor(private genericConfigUtilsService: GenericConfigUtilsService) {}
+  closeActiveModal(): void {
+    const translation = this.getDisplayOnly()
+      ? this.translation.translate('configurator.header.displayConfiguration')
+      : this.translation.translate('configurator.header.editConfiguration');
+
+    translation
+      .pipe(take(1))
+      .subscribe((reason) => this.modalService.closeActiveModal(reason));
```

Comment:
Just a question: Why do we need to provide this reason?

Reply:
As per our conversation Larisa: Please remove localization here

Reply:
Done